sirepo/exporter.py

```diff
@@ -60,12 +60,21 @@ def _create_zip(sim, out_dir, qcall):
     data = simulation_db.open_json_file(sim.type, sid=sim.id, qcall=qcall)
     simulation_db.update_rsmanifest(data)
     data.pkdel("report")
+    if hasattr(sim.template, "add_fdir"):
+        sim.template.add_fdir(data, sim)
```

Comment:
I don't think this is needed if the fdir part is added in python_source_for_model in template.srw.